core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1586,6 +1586,11 @@ class EMConfig {
     @Experimental
     var instrumentMR_NET = false
 
+    @Cfg("Execute instrumentation for method replace with category OPENSEARCH." +
+            " Note: this applies only for languages in which instrumentation is applied at runtime, like Java/Kotlin" +
+            " on the JVM.")
+    var instrumentMR_OPENSEARCH = true
```

Comment:
this should be marked as `@Experimental`. also, it should be `false` by default. for E2E tests, can activate them there (however note must be done before classes are loaded)